README.md

```diff
@@ -39,15 +39,18 @@ The column chart uses an *array* of crossfilter groups to display different type
 * `xAxis` (Object): `domain` (Array, probably of `moment` objects) and `ticks` (number of ticks to show)
 
 ##### Optional parameters
-* `yAxis` (Object): `domain` (Array, probably of numbers) (optional), `ticks` (number of ticks to show), and `formatter` (function to apply to every y-axis value for tick display)
+* `yAxis` (Object): `domain` (Array, probably of numbers) (optional): autofits bars, `ticks` (number of ticks to show), and `formatter` (function to apply to every y-axis value for tick display), `bottomLabelPosition` (boolean): whether to display labels on the top, or on the bottom of x-axis.
 * `type` (String) (defaults to `GROUPED`):
     * `GROUPED` is for "ordinary" data and is most likely what should be used if there is only one crossfilter group.
     * `LAYERED` is for "overlapping data: e.g. there are 10 fruits, 6 of which are citrus, 3 of which are oranges. If this chart is `LAYERED`, the `series` option tells the chart how to format the bars (hatching).
     * `STACKED` creates a stacked bar chart: e.g. there are 10 fruits, 20 vegetables, and 15 meats and we want to display them on top of each other.
 * `series` (Array):  Each object in the array has properties:
     * `title`: the name of `this.get('group')` at the same index (used in tooltip creation)
     * `hatch`: `pos` (for a hatch from bottom left to top right), `neg` (for a hatch from top left to bottom right), or `false` (for no hatch)
-* `showMaxMin` (boolean): whether or not to show max/min indicators for the maximum and minimum values of one of the groups on the column chart
+* `LabelOptions`(Object): display options for labels on top of bars/columns.
+    * `showMaxMin` (boolean): whether or not to show max/min indicators for the maximum and minimum values of one of the groups on the column chart.
+    * `showDataValues` (boolean): whether or not to display the y-value of each point above each bar.
+    * `labelCollisionResolution` (boolean): runs collision detection algorithm to determine if a label is too wide, and skips next labels accordingly.
```

Comment:
```suggestion
    * `labelCollisionResolution` (string): runs collision detection algorithm to determine if a label is too wide, and skips next labels accordingly.
```

All uses of this below are as a string, so I assume this needs updated to reflect that. Probably should mention which modes ['auto'] are supported as well